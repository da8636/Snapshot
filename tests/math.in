SET a 5 4 7
MIN a
MAX a
SUM a
LEN a
BYE
