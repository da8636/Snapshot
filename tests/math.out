ok

4

7

16

3

bye
